### PR TITLE
Add content to About and Contact pages

### PR DESCRIPTION
**Pull request by reginafcompton**
# Overview

This PR adds content provided by CSH to the About and Contact Us pages.

It handles issue #95.

The changes that could use an extra eyes appear on [the About page.](https://github.com/datamade/coordinated-entry-screening/pull/95/files#diff-3ba1a83b12d0ed1e8dea928f03013208R1)
* does the about page render as expected?
* does the html/bootstrap attributes seem sane?
* do the logos at the bottom make sense, esp., hiding them in mobile view (or should I stack them)?

# Testing

Pull down the changes, and then:

```
# activate your virtualenv
workon coordinated-entry-screening

pip install -r requirements.txt --upgrade

python manage.py runserver
```

Visit 1.27.0.0.1:8000/about.

### CODE REVIEW

File: templates/ces_client/about.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        <p>Coordinated Entry is the front door to connect people facing homelessness to housing and supports in an accessible, equitable, transparent manner. This allows all partners to collaborate to improve access and outcomes for people without permanent housing while delivering trauma informed services.</p>
+        <p>People facing homelessness are asked to complete a Housing Assessment and linked to housing, if eligible, as opportunities become available.</p>
+
+        <h5 class='mt-5'><strong>Shelter Access in Cook County Suburbs</strong></h5>
+        <p>View shelter options and next steps <a href="http://www.suburbancook.org/emergency" target="_blank">by clicking here</a>.</p>
+
+        <h5 class='mt-5'><strong>Shelter Access In Chicago</strong></h5>
+        <p>Please visit any hospital emergency room or police station in Chicago, or call 311 and request transportation to shelter.</p>
+        <p>
+            <strong><i class="fas fa-user"></i> Single individuals can request shelter in person at the Chicago Garfield Service Center.</strong><br>
```
Comment (jmithani) [inline templates/ces_client/about.html]: i would suggest changing these to side-by-side cols in a row. i would have the icon on top of the text too.

File: templates/ces_client/contact.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{% extends "base.html" %}
+{% load static %}
+{% load ces_client_extras %}
+{% block title %}Connect{% endblock %}
```
Comment (jmithani) [inline templates/ces_client/contact.html]: Make sure to change the titles!

File: templates/ces_client/contact.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            <div class='col'>
+                <h3 class='mt-4'>Connect to End Homelessness</h3>
+                <h4>A unified community response</h4>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class='row'>
+    <div class='col-10 offset-1 mb-3 pb-3'>
```
Comment (jmithani) [inline templates/ces_client/contact.html]: Let's try centering this text. It looks a bit wonky on desktop—too heavy visually on the left.

File: templates/ces_client/about.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        <h5 class="mt-5 mb-3"><strong>About the Bravely Engineered Navigator (B.E.N.)</strong></h5>
+        <div class='row'>
+            <div class='col'>
+                <p>B.E.N. represents a robot who knows a lot about the homeless experience. He helps personalize the survey-taking process. <a href="https://datamade.us/" target="_blank">DataMade (a civic technology company in Chicago)</a> designed B.E.N. and "Connect to End Homelessness" in partnership with the <a href="https://www.csh.org" target="_blank">Corporation for Supportive Housing (a non-profit that advances supportive housing solutions)</a>.</p>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class='row d-none d-sm-block'>
```
Comment (jmithani) [inline templates/ces_client/about.html]: The logos looked like they could be scaled appropriately in one row. If not, I'd keep them in stacked.

**Review by jmithani — CHANGES_REQUESTED** [review]
This looks really good! I'm so excited for this project to launch.

A few extra comments:
- On mobile, the menu doesn't seem as intuitive. Since we only have three links, is it possible to [expand them instead](https://getbootstrap.com/docs/4.3/components/navbar/#responsive-behaviors)?
- For bold links, have the CSS switch to one of the [bold fonts](https://fonts.google.com/specimen/Libre+Franklin) from the Libre Franklin typeface. Using `<strong>` and `<bolder>` don't always communicate the differences in weight. In general, it's good to skip a font weight (e.g. pairing Regular, Semi-bold, and Extra-bold.

Let me know what you think of my suggestions.

File: templates/ces_client/contact.html (hunk 2 above)
Comment (reginafcompton) [reply templates/ces_client/contact.html]: Oh, boy....

File: templates/ces_client/contact.html (hunk 3 above)
Comment (reginafcompton) [reply templates/ces_client/contact.html]: I am not so sure. I looks odd, both ways, given that we do not have much content for this page.

**Comment by reginafcompton** [comment]
Thanks for this wonderfully helpful feedback! I believe that I implemented the feedback (with the exception of centering the text on the Contact page). Let me know what you think!

File: templates/ces_client/contact.html (hunk 3 above)
Comment (jmithani) [reply templates/ces_client/contact.html]: That works for me!

File: templates/ces_client/about.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        <p>Coordinated Entry is the front door to connect people facing homelessness to housing and supports in an accessible, equitable, transparent manner. This allows all partners to collaborate to improve access and outcomes for people without permanent housing while delivering trauma informed services.</p>
+        <p>People facing homelessness are asked to complete a Housing Assessment and linked to housing, if eligible, as opportunities become available.</p>
+
+        <h5 class='mt-5 font-weight-bold'>Shelter Access in Cook County Suburbs</h5>
+        <p>View shelter options and next steps <a href="http://www.suburbancook.org/emergency" target="_blank">by clicking here</a>.</p>
+
+        <h5 class='mt-5 font-weight-bold'>Shelter Access In Chicago</h5>
+        <p>Please visit any hospital emergency room or police station in Chicago, or call 311 and request transportation to shelter.</p>
+        <div class='row'>
+            <div class='col'>
```
Comment (jmithani) [inline templates/ces_client/about.html]: I was imagining something like this, with the icon centered and larger:
Screen Shot 2019-07-03 at 10 20 02 AM

Having it in line with the text is difficult to read.

File: templates/ces_client/about.html [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+<div class='row'>
+    <div class='col-8 offset-2'>
+        <div class='row'>
+            <div class='col pt-3'><a class="img-fluid float-left mr-0" href="https://www.csh.org" target="_blank"><img class="img-width-sm" src="{% static 'images/CSH_Logo.svg' %}"></a></div>
+            <div class='col'><a class="img-fluid float-left mr-0" href="{% url 'index' %}"><img class="img-width-sm" src="{% static 'images/ben-icon-03.svg' %}" width='100'></a></div>
+            <div class='col pt-3'><a class="img-fluid float-left mr-0" href="https://datamade.us/" target="_blank"><img class="img-width-md" src="{% static 'images/datamade_logo.png' %}" width='180'></a></div>
+        </div>
+    </div>
+</div>
```
Comment (jmithani) [inline templates/ces_client/about.html]: This is what it looks like on mobile now:
Screen Shot 2019-07-03 at 10 23 05 AM

I'd suggest stacking them into one column (i.e. one on top of each other) or reducing the space between each logo. If this is finicky, reverting to hiding them is okay.

**Review by jmithani — CHANGES_REQUESTED** [review]
I added screenshots to clarify my comments. The font looks SO GOOD!

File: templates/ces_client/about.html (hunk 5 above)
Comment (reginafcompton) [reply templates/ces_client/about.html]: Got it! Thanks for the image, too.

File: templates/ces_client/about.html (hunk 6 above)
Comment (reginafcompton) [reply templates/ces_client/about.html]: Got them to stack!

**Review by jmithani — APPROVED** [review]
AMAZING thank you so much for seeing this through!

**Pull request merged**